components/checkbox/README.md

```diff
@@ -8,66 +8,67 @@ author: orinevares
 
-## Don't Use This for:
-* If a user can only select one option from a list, use a radio button instead.
-
 ## Rationale
 Our checkboxes are larger than most browser defaults to allow for greater visibility and larger target areas for touchscreen devices. Research has shown that users resort to only clicking the checkbox and not the associated text, therefore a larger target area is necessary. Simply enlarging browser defaults proved to be problematic on many web browsers and the custom design should be used.
-Based on research done by Gov.UK: https://designnotes.blog.gov.uk/2016/11/30/weve-updated-the-radios-and-checkboxes-on-gov-uk/
+Based on [research](https://designnotes.blog.gov.uk/2016/11/30/weve-updated-the-radios-and-checkboxes-on-gov-uk/) done by Gov.UK.
 
 ## Behaviour
 1. Users should be able to tap or click on either the checkbox or the label to select the option.
+2. Do not preselect options, this signals a biased result.
 
 ## Accessibility
-This checkbox has been built according to WCAG 2.0 AA standards and has been successfully tested for the following:
-1.	Perceivable
-* Alternative text is provided for non-text content (WCAG 1.1.1)
-*	Content can be presented in different ways without losing information or structure (WCAG 1.3)
-*	Colour is not used as the only visual means of conveying information (WCAG 1.4.1)
-*	Visual presentation of text and images has a contrast ratio of at least 4.5:1 (WCAG 1.4.3)
-*	Text can be resized without assistive technology up to 200% without loss of content or functionality (WCAG 1.4.4)
-
-2.	Operable
-*	All functionality of the content is operable through a keyboard interface (WCAG 2.1.1)
-*	A mechanism is available to bypass blocks of content that are repeated on multiple web pages (WCAG 2.4.1)
-*	Headings and Labels describe topic or purpose (WCAG 2.4.6)
-*	Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible (WCAG  2.4.7)
-
-3.	Understandable
-*	The default human language of each web page can be programmatically determined (WCAG 3.1.1)
-*	Navigational mechanisms that are repeated on multiple web pages occur in the same relative order each time they are repeated (WCAG 3.2.3)
-
-4.	Robust
-*	For all user interface components, the name and role can be programmatically determined (WCAG 4.1.2)
-
-## Design with This Component
+This checkbox has been built according to [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/) standards and all government services should strive to meet this level.  This component successfully includes the following accessibility features:
+
+### Screenreaders
+* Proper ARIA and ALT labels are included
+* Language tag has been set to english
+* List items and checkbox status are labelled properly
+
+### Colour Contrast
+* Contrast ratios are at least 7:1 for normal text and 4.5:1 for large text
+
+### Tab Navigation
```

Comment:
Can we change this to keyboard navigation instead of Tab navigation?

Reply:
Sure, that makes more sense.